### PR TITLE
Add `aesthetic` spinner

### PR DESCRIPTION
**Pull request by Geshii**
Hi, I added some new styles for the project, will add some other styles later maybe ^^', hope it'll help.

### CODE REVIEW

File: spinners.json [hunk 1]
```diff
@@ -1278,5 +1278,25 @@
 			"βββββρβ",
 			"ββββββρ"
 		]
+	},
+		"fish": {
+		"interval": 150,
+		"frames": [
+			">))'>",
+			"<'((<"
```
Comment (sindresorhus) [inline spinners.json:1286]: I don't think this is clear enough as being as fish when animated. To me, it just looks like noise. I would drop this one.

File: spinners.json [hunk 2]
```diff
@@ -1278,5 +1278,25 @@
...
+		]
+	},
+			"futurebar": {
```
Comment (sindresorhus) [inline spinners.json:1289]:
```suggestion
	"futurebar": {
```

File: spinners.json [hunk 3]
```diff
@@ -1278,5 +1278,25 @@
...
+		"interval": 80,
+		"frames": [
+			"▰▱▱▱▱▱▱",
+			"▰▰▱▱▱▱▱",
+			"▰▰▰▱▱▱▱",
+			"▰▰▰▰▱▱▱",
+			"▰▰▰▰▰▱",
```
Comment (sindresorhus) [inline spinners.json:1296]: This is missing a square.

File: spinners.json (hunk 2 above)
Comment (sindresorhus) [reply spinners.json:1289]: I think it would be better to use a descriptive name.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: spinners.json (hunk 3 above)
Comment (Geshii) [reply spinners.json:1296]: will change that

File: spinners.json (hunk 2 above)
Comment (Geshii) [reply spinners.json:1289]: like, aesthetic ?

File: spinners.json (hunk 1 above)
Comment (Geshii) [reply spinners.json:1286]: okayy

**Comment by Geshii** [comment]
@sindresorhus fixed ^^'

**Pull request merged**